### PR TITLE
Add pbr versioning

### PR DESCRIPTION
**Pull request by bnelsj**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Brad Nelson <bnelsj@gmail.com>
+Brad Nelson <brad@phasegenomics.com>
+EC2 Default User <ec2-user@ip-172-31-25-134.us-west-2.compute.internal>
```
Comment (shawnpg) [inline AUTHORS]: Is it necessary to have the default ec2-user be an author? In theory anyone can be that user

File: Readme.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The most informative Hi-C reads are the ones that are long-distance contacts, or
 Hi-C connectivity drops off in approximately a power-law with increasing linear sequence distance. Consequently, one expects Hi-C reads to follow a characteristic distribution, wherein there is a spike of many read pairs at distances close to zero, which drops off smoothly (in log space) with increasing distance. If there are odd spikes or discontinuities, or if there are few long-distance contacts, there may be a problem either with the library or the assembly.
 
 ## Dependencies
-* python2.7 or python3.6 (mostly tested using 2.7.10 and 3.6.5)
+* python 2.7 or python >3.4 (mostly tested using 2.7.10 and 3.6.5)
```
Comment (shawnpg) [inline Readme.md]: Should we leave this at 3.6 if we haven't tested earlier ones?

File: AUTHORS (hunk 1 above)
Comment (shawnpg) [inline AUTHORS]: Is it necessary to have the default ec2-user be an author? In theory anyone can be that user

Comment (bnelsj) [reply AUTHORS]: It was created automatically by pbr, so there must be hotfix commits somewhere with that username.

Comment (shawnpg) [reply AUTHORS]: OK

**Pull request merged**